compiler/rustc_trait_selection/src/traits/select/mod.rs

```diff
@@ -123,8 +124,8 @@ pub struct SelectionContext<'cx, 'tcx> {
 }
 
 // A stack that walks back up the stack frame.
-struct TraitObligationStack<'prev, 'tcx> {
-    obligation: &'prev TraitObligation<'tcx>,
+struct PolyTraitObligationStack<'prev, 'tcx> {
```

Comment:
i think we should keep this as `TraitObligationStack`, the `Poly` doesn't add any meaningful info imo and makes the code harder to read imo

Reply:
No this is just because I was stupid and used ctrl+f to replace names, and I didn't see this was edited too 🤦